controller/search.js

```diff
@@ -102,11 +102,15 @@ function setup( apiConfig, esclient, query, should_execute ){
           const messageParts = [
```

Comment:
Oh, is `messageParts` actually unused here? This looks like our old (pre-JSON) logging format that is hard to parse and doesn't really do anything useful anymore.

Reply:
Hmm.. good catch, I added a second commit to remove that code

Reply:
Thanks. Its my fault for leaving it in long ago :)